conda_gitenv/__init__.py

```diff
@@ -1,7 +1,6 @@
-__version__ = '0.1.0'
```

Comment:
Note, this fixes https://github.com/SciTools/conda-gitenv/pull/14#discussion_r46541390.